tests/units/test_lib.rb

```diff
@@ -13,7 +14,16 @@ def setup
     set_file_paths
     @lib = Git.open(@wdir).lib
   end
-  
+
+  def test_fetch_unshallow
+    Dir.mktmpdir("ruby-git-shallow") do |dir|
```

Comment:
You do not need to include/require that.

We have `in_temp_dir` inside our helper that can do this work for you.  You can see an example in https://github.com/ruby-git/ruby-git/blob/master/tests/units/test_stashes.rb#L11

Reply:
updated